tests/e2e/common.go

```diff
@@ -266,12 +265,12 @@ func streamPodLogs(ctx context.Context, client clientgo.Interface, namespace, po
 				return true, nil
 			}
 
-			fmt.Printf("Got error %v, retrying.\n", err)
 			return false, nil
 		})
 
 		if err != nil {
-			log.Fatalf("Failed to stream pod %q logs: %v", podName, err)
+			fmt.Printf("Failed to stream pod %q logs: %v", podName, err)
```

Comment:
Maybe we can use some logger library and log it as error, for example we are already using `logrus`/`zerolog` in this project.
But can be as an improvement in upcoming PR.

Reply:
Thanks Chandan! Let me mark that as TODO.